.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+orbs:
+  android: wordpress-mobile/android@1.0
+
+version: 2.1
+jobs:
+  BuildAndTest:
+    executor:
+      name: android/default
+      api-version: "30"
+    steps:
+      - checkout
+      - android/restore-gradle-cache
+      - run:
+          command: ./gradlew --stacktrace assembleRelease
+      - run:
+          command: ./gradlew --stacktrace testRelease
+      - run:
+          command: ./gradlew --stacktrace publishToMavenLocal
```

Comment:
Sorry for change after requesting review. I've just added this step as checking of publishing to maven local works fine is a useful check in context of migrating to `Jitpack` #56

Reply:
I am not sure how I feel about this to be honest. It feels a bit out of context to me and a little unnecessary. I think the `assemble` step should be enough and when/if we start using Jitpack for this repo, we should get warned of any errors. We don't have this step for other projects that uses Jitpack and we didn't have any issues in them this step would help with. Finally, we might not even use Jitpack, it's just a probability.

What do you think @loremattei @jkmassel?

Reply:
> we should get warned of any errors

I don't know how `Jitpack` warns about problems but running `publishToMavenLocal` on CI would prevent us from merging PR that would break on `Jitpack` before any client could use it.

E.g. we've got this scenario at this moment. `publishToMavenLocal` does not work (reason is described in #56 ) but both `assemble` and `test` tasks works fine.

I'm ok with removing this check, I thought it might be a good idea to have extra safety on that and I don't see any negative outcomes.

Reply:
After spending some time trying to figure out why Jitpack is being weird, I can see the value in having this task here. Let's give it a shot.

Can we add a comment explaining this is for testing Jitpack integration?

Reply:
I've added names to steps instead of writing comment in 69f04d2a62930370ae6c8ebf6074fe8d0e04c8f8. This way it's visible already in CircleCI logs that this step is about asserting Jitpack build correctness. Does it work for you?

Reply:
This looks great, thank you!